client/state/test/initial-state.js

```diff
@@ -796,9 +795,22 @@ describe( 'loading stored state with dynamic reducers', () => {
 	const withKeyPrefix = ( keyPrefix ) => {
 		const keyPrefixRe = new RegExp( `^${ keyPrefix }:` );
 		return withPersistence( ( state = {} ) => state, {
-			serialize: ( state ) => mapKeys( state, ( value, key ) => `${ keyPrefix }:${ key }` ),
+			serialize: ( state ) =>
+				Object.fromEntries(
+					Object.entries( state ).reduce( ( acc, [ key, value ] ) => [
```

Comment:
We don't seem to be using the `acc`. Did you mean `map`? 

```suggestion
					Object.entries( state ).map( ( [ key, value ] ) => [
```

Reply:
Should be fixed with https://github.com/Automattic/wp-calypso/pull/81560/commits/953caf72714237f0e3b7f952c24080560b0ea34a